client_plugin/vmdk_plugin/main_windows.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A VMDK Docker Data Volume plugin for Windows
+package main
+
+import (
+    "github.com/kardianos/service"
+    log "github.com/Sirupsen/logrus"
+)
+
+var logger service.Logger
+
+type program struct{}
```

Comment:
We should probably add a comment explaining the need to have this extra bit for Windows.